tests/credential_plugins_test.py

```diff
@@ -131,6 +132,76 @@ def test_hashivault_handle_auth_not_enough_args():
         hashivault.handle_auth()
 
 
+def test_aws_assumerole_with_accesssecret(monkeypatch):
```

Comment:
While on it, please add typing
```suggestion
def test_aws_assumerole_with_accesssecret(monkeypatch: pytest.MonkeyPatch) -> None:
```